AElf.OS.Network.Grpc/GrpcPeerPool.cs

```diff
@@ -76,7 +99,12 @@ private async Task<bool> Dial(string address)
                         
                 var client = new PeerService.PeerServiceClient(channel);
                 var hsk = await BuildHandshakeAsync();
-                        
+                
+                if (channel.State != ChannelState.Connecting)
```

Comment:
why "channel.State != ChannelState.Connecting"?

Reply:
This is because sometimes (especially on the CI) the socket goes into the TRANSIENT_FAILURE state, but this can be normal and it only stays in this state for a very short time, but if you try to call an RPC method it will throw. So when I create the channel and IF it's not in CONNECTING state, I give it some time to sort itself out.